content/docs/get-started/gcp/begin.md

```diff
@@ -50,23 +50,48 @@ Next, we'll install the required language runtime.
 {{< install-dotnet >}}
 {{% /choosable %}}
 
-Next, we'll configure GCP.
+Finally, configure Pulumi with Google Cloud.
 
-### Configure GCP
+### Configure Pulumi to access your Google Cloud account
 
-The [Pulumi Google Cloud Platform Provider]({{< relref "/docs/intro/cloud-providers/gcp" >}}) needs to be configured with Google credentials
-before it can be used to create resources.
+Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user or service account that has **Programmatic access** with rights to deploy and manage resources handled through Pulumi. 
```

Comment:
```suggestion
Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user or service account that has **Programmatic access** with rights to deploy and manage resources. 
```

... only a suggestion, since "with rights to deploy ... resources handled through Pulumi" sounds a bit like the account needs rights to do something Pulumi-specific, which isn't the case. (It just needs rights to deploy and manage GCP resources.)  

Also, would it be worth noting that this is specifically a Google Cloud IAM user or service account, and maybe link to a relevant doc?

Reply:
Incorporated the change in the first suggestion, but I'm not sure we need to link to the Google Cloud IAM docs or add the qualifier than this is a Google Cloud IAM user (as opposed to AWS) as Google uses that acronym as well.